build-logic/build-platform/build.gradle.kts

```diff
@@ -18,7 +18,6 @@ val kotlinVersion = providers.gradleProperty("buildKotlinVersion")
 dependencies {
     constraints {
         api("com.gradle:gradle-enterprise-gradle-plugin:3.11")
-        api("com.gradle.enterprise:test-distribution-gradle-plugin:2.3.5") // Sync with `settings.gradle.kts`
```

Comment:
I think the comment should be preserved and put on the line above.

Reply:
done